Fixed some problem and corrected to conform for latest version of st2(v2.8)

userlocalhost opened the pull request:
I'm sorry for being late to correct after receiving some reports (#11, #14). I fixed some problem and corrected to conform for latest version of st2.

This patch includes following changes.

* Changed MediaType parameter of StackStorm to access through the API
  endpoints instead of accessing directly to each service processes.
* Corrected "st2_dispatcy.py" to conform to be able to communicate with
  up to date st2api (v2.8).
* Changed to be able to overwrite Zabbix configuration for StackStorm by
 'register_st2_config_to_zabbix.py'.
* Fixed typos in README

To confirm this changes in your environment, please execute `register_st2_config_to_zabbix.py` to reset configuration in Zabbix for StackStorm.
```
$ cd stackstorm-zabbix
$ tools/register_st2_config_to_zabbix.py -z http://zabbix-host/zabbix -u Admin -p zabbix
```
**CAUTION**
After execute this command, your configuration of MediaType and Action setting in Zabbix will be overwritten with initial value like this.
<img width="903" alt="2018-07-30 22 47 43" src="https://user-images.githubusercontent.com/469934/43401542-5ca6cbb2-944b-11e8-8ff4-4b5344bff330.png">

nmaludy commented on CHANGELOG.md:
```diff
@@ -1,5 +1,13 @@
 
+## 0.1.7
+
+- Changed MediaType parameter of StackStorm to access through the API endpoints instead of accessing directly to each service processes.
```
Should probably add a note in the CHANGELOG about the new parameters for st2_dispatch.py since it changed from --st2-host to --st2-api-url and --st2-auth-url

nmaludy replied:
Should probably add a note in the CHANGELOG about the new parameters for st2_dispatch.py since it changed from --st2-host to --st2-api-url and --st2-auth-url

nmaludy commented on README.md:
````diff
@@ -13,7 +13,7 @@ First of all, you can install this pack as below
 $ st2 pack install zabbix
 ```
````
Should update the screenshots in here with the new parameters for the dispatch script.

userlocalhost replied:
I updated screenshot and descriptions for the `st2_dispatch.py` by bf6777e and 809cb09.

userlocalhost commented:
Thank you very much for your prompt review, @nmaludy. I should be able to fix the points that you mentioned and the problem that CI noticed.

userlocalhost commented:
I updated followings after the first commit of this PR.

- Fixed problems which flake8 noticed.
- Updated CHANGELOG to specify this patch is backword incompatible and wrote information to upgrade from previous version.
- Moved images for README from other repository.
- Updated images and expression in README conform to this patch.

LindsayHill commented:
Looks pretty good now. @nmaludy you happy with the changes?

nmaludy commented:
Looks good to me!

LindsayHill commented:
Fixes #11, #14
